### PR TITLE
Get offline batch inference details using task API in m

### PR DESCRIPTION
**Pull request by rbhavna**
### Description
Add documentation for getting offline batch inference details using task API in ml-commons

### Issues Resolved
closes https://github.com/opensearch-project/documentation-website/issues/8210

### Version
2.17

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _ml-commons-plugin/api/model-apis/batch-predict.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ POST /_plugins/_ml/models/<model_id>/_batch_predict
 
 ## Prerequisites
 
-Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request:
+Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` paramter supports cancelling the batch job that is running on OpenAI:
```
Comment (natebower) [inline _ml-commons-plugin/api/model-apis/batch-predict.md:34]:
```suggestion
Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` paramter supports canceling the batch job running on OpenAI:
```

File: _ml-commons-plugin/api/model-apis/batch-predict.md [hunk 2]
```diff
@@ -123,45 +139,87 @@ POST /_plugins/_ml/models/lyjxwZABNrAVdFa9zrcZ/_batch_predict
...
-    }
-  ]
+  "task_id": "KYZSv5EBqL2d0mFvs80C",
+  "status": "CREATED"
 }
 ```
 
-For the definition of each field in the result, see [OpenAI Batch API](https://platform.openai.com/docs/guides/batch). Once the batch inference is complete, you can download the output by calling the [OpenAI Files API](https://platform.openai.com/docs/api-reference/files) and providing the file name specified in the `id` field of the response.
\ No newline at end of file
+To check the status of the batch job operation, provide the task ID to the [Tasks API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/tasks-apis/get-task/). You can find the job details in the `remote_job` field in the task. Once the prediction is complete, the task `state` changes to `COMPLETED`.
```
Comment (natebower) [inline _ml-commons-plugin/api/model-apis/batch-predict.md:151]: "batch job" => "batch predict"?

**Review by natebower — COMMENTED** [review]
@kolchfa-aws @rbhavna Just a couple minor changes. Thanks!

File: _ml-commons-plugin/api/model-apis/batch-predict.md (hunk 2 above)
Comment (kolchfa-aws) [inline _ml-commons-plugin/api/model-apis/batch-predict.md:151]:
```suggestion
To check the status of the batch predict job, provide the task ID to the [Tasks API]({{site.url}}{{site.baseurl}}/ml-commons-plugin/api/tasks-apis/get-task/). You can find the job details in the `remote_job` field in the task. Once the prediction is complete, the task `state` changes to `COMPLETED`.
```

Comment (kolchfa-aws) [reply _ml-commons-plugin/api/model-apis/batch-predict.md]: Reworded.

**Review by kolchfa-aws — APPROVED** [review]
Thanks, @rbhavna! LGTM

File: _ml-commons-plugin/api/model-apis/batch-predict.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ POST /_plugins/_ml/models/<model_id>/_batch_predict
 
 ## Prerequisites
 
-Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request:
+Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` paramter supports canceling the batch job running on OpenAI:
```
Comment (Zhangxunmt) [inline _ml-commons-plugin/api/model-apis/batch-predict.md:34]: The optional action_type parameter allows for checking the status or canceling a batch job running on OpenAI.

**Review by Zhangxunmt — APPROVED** [review]

File: _ml-commons-plugin/api/model-apis/batch-predict.md (hunk 3 above)
Comment (rbhavna) [reply _ml-commons-plugin/api/model-apis/batch-predict.md:34]: Can we have something like this: 
Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The action_type `cancel_batch_predict` is optional and it supports canceling the batch job running on OpenAI:

Comment (kolchfa-aws) [inline _ml-commons-plugin/api/model-apis/batch-predict.md:34]:
```suggestion
Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` parameter supports canceling the batch job running on OpenAI:
```

File: _ml-commons-plugin/api/model-apis/batch-predict.md [hunk 4]
```diff
@@ -31,7 +31,13 @@ POST /_plugins/_ml/models/<model_id>/_batch_predict
 
 ## Prerequisites
 
-Before using the Batch Predict API, you need to create a connector to the externally hosted model. For example, to create a connector to an OpenAI `text-embedding-ada-002` model, send the following request. The optional `action_type` parameter supports canceling the batch job running on OpenAI:
+Before using the Batch Predict API, you need to create a connector to the externally hosted model.  For every action, specify the  `action_type` parameter that describes the action:
```
Comment (natebower) [inline _ml-commons-plugin/api/model-apis/batch-predict.md:34]: There are extra spaces before "For every" and after "specify the". I would say "For each action".

**Review by natebower — APPROVED** [review]
LGTM with noted comments.

**Pull request merged**